src/main/java/ru/urururu/sanity/cpp/tools/Language.java

```diff
@@ -25,4 +22,21 @@
     public Set<String> getExtensions() {
         return extensions;
     }
+
+    private static final Map<String, Language> byExtensions = new HashMap<>();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)